_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_cta-form.scss

```diff
@@ -62,34 +62,3 @@
         width: 100%;
     }
 }
-
-.region.region-content {
```

Comment:
I moved this to _form.scss so that  the _cta-form.scss will be specific to the assembly type.